.openpublishing.publish.config.json

```diff
@@ -44,6 +44,12 @@
       "url": "https://github.com/Microsoft/templates.docs.msft",
       "branch": "master",
       "branch_mapping": {}
+    },
+    {
+      "path_to_root": "docs/samples",
```

Comment:
I'd probably configure the same way we have our samples configured in docs (but with a different repo)
https://github.com/dotnet/docs/blob/master/.openpublishing.publish.config.json#L90